aspnet/web-forms/overview/deployment/advanced-enterprise-web-deployment/deploying-membership-databases-to-enterprise-environments.md

```diff
@@ -13,6 +13,9 @@ msc.type: authoredcontent
 
 by [Jason Lee](https://github.com/jrjlee)
 
+> [!NOTE]
+> Since this document was written, the in-box ASP.Net Membership providers have been superseded by '[ASP.Net Identity](https://learn.microsoft.com/en-us/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity)'. Please update applications to use the [ASP.Net Identity](https://learn.microsoft.com/en-us/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity) platform instead of the in-box Membership providers.
```

Comment:
```suggestion
> Since this article was written, the in-box ASP.NET Membership providers have been superseded by '[ASP.NET Identity](../../../../identity/overview/getting-started/introduction-to-aspnet-identity.md)'. Please update applications to use the [ASP.NET Identity](../../../../identity/overview/getting-started/introduction-to-aspnet-identity.md) platform instead of the in-box Membership providers.
```